Sources/armory/logicnode/GetCameraFovNode.hx

```diff
@@ -11,7 +11,7 @@ class GetCameraFovNode extends LogicNode {
 	override function get(from: Int): Dynamic {
 		var camera: CameraObject = inputs[0].get();
 
-		if (camera == null) return 0.0;
+		if (camera == null) return null;
```

Comment:
I'm not sure about this. `null` values are often forgotten and lead to errors, also they're sometimes problematic in Haxe when targeting static platforms (read about nullability in Haxe). If someone get's a value of `0.0` I think it's clear that the camera couldn't be found.

If you leave it on `null`, this needs to be documented.